test/controllers/auth/AuthActionSpec.scala

```diff
@@ -266,4 +266,44 @@ class AuthActionSpec extends FreeSpec with MustMatchers with MockitoSugar with O
         s"Some(TrustedHelper(principalName,attorneyName,returnUrl,$fakePrincipalNino))")
     }
   }
+
+  "A user with a SCP Profile Url must include a redirect uri back to the home controller" in {
```

Comment:
What would happen if you provided an invalid url?

Reply:
I've put together a fix for it the URL has a scheme as a way to check for an absolute URL, but I'm 100% sure that the SCP Profile Urls in prod are absolute or relative.

Reply:
I've had this confirmed by #team-scp-si. It's always absolute

Reply:
We should assume they can be both

Reply:
Thats fine the way it is then